### ISSUE TITLE
After changing Dynamic Text size, site title in header did not restore to original size 

### ISSUE THREAD

**Issue by startuptester** [wordpress-mobile/WordPress-iOS#16375]
### Expected behavior
Site title should adjust based on changes to Dynamic Text (now called Larger Text)

### Actual behavior
Even after I logged out and back in, it's still large. Force closing restored it properly. 

![IMG_3367](https://user-images.githubusercontent.com/66104034/116138174-b578e580-a691-11eb-80f3-759c7bd4491e.PNG)

### Steps to reproduce the behavior
1. Change system font size to something noticeably larger
2. View in app to see Site Title is large on my Site tab
3. Restore original font size
4. View My Site tab, still large title.

**Comment by diegoreymendez** [comment]
@startuptester - I'm attaching a GIF showing my tests, in which I'm not being able to reproduce the reported issue.

Can I ask you to verify if the steps I'm taking are the same steps you took to test this issue?  Thanks!

![largeText](https://user-images.githubusercontent.com/1836005/116993543-eaabb600-acd7-11eb-9fdc-bfd4335aa7d4.gif)

**Comment by startuptester** [comment]
Thanks @diegoreymendez, I was unable to reproduce this in the latest beta (17.3) as well.
